### PR TITLE
Notes section

### PR DESCRIPTION
**Pull request by Manpreet-hub**
**Notes section features:-**

- Add a Note
- Remove a Note
chrome_xE9FKu4209

### CODE REVIEW

File: src/components/TextEditor/TextEditor.jsx [hunk 1]
```diff
@@ -31,15 +26,13 @@ const TextEditor = () => {
...
-            setNote({ ...note, title: e.target.value })
+            setNote({ ...note, title: e.target.value, id: uuidv4() })
           }
         />
         <ReactQuill
           theme="snow"
-          value={note}
-          // onChange={inputHandler}
-          // onChange={(e) => setNote({ ...note, content: e.target.value })}
+          value={note.content}
```
Comment (abhijeetekad) [inline src/components/TextEditor/TextEditor.jsx:34]: Remove commented code

**Review by abhijeetekad — APPROVED** [review]
Note title is remaining same after adding the note. Please check that otherwise good to merge.

**Review by Richa-rani02 — APPROVED** [review]
image
close button is going out of container .You can check this

File: src/components/TextEditor/TextEditor.jsx (hunk 1 above)
Comment (Manpreet-hub) [reply src/components/TextEditor/TextEditor.jsx:34]: @abhijeetekad  title is updating after adding the note. Please check it again. 
@Richa-rani02  Thanks for pointing out, will work on this in the next PR.

Comment (abhijeetekad) [reply src/components/TextEditor/TextEditor.jsx:34]: Yes .. just saw that .. my bad

**Pull request merged**